types/mparticle-web-sdk/index.d.ts

```diff
@@ -22,7 +22,21 @@ export const CommerceEventType: {
     PromotionView: number;
 };
 
-export const config: {
+export let config: {
+    isDevelopmentMode?: boolean,
+    identifyRequest?: identifyRequest,
+    identityCallback?: () => any,
```

Comment:
You should define this interface above:

```
interface Callback {
    () : void;
}
```

So that you can have this return type as `Callback`.